ios/FluentUI/People Picker/PersonaBadgeViewDataSource.swift

```diff
@@ -11,8 +11,9 @@ import UIKit
 open class PersonaBadgeViewDataSource: BadgeViewDataSource {
     @objc open var persona: Persona
 
-    @objc public init(persona: Persona, style: BadgeView.Style = .default, size: BadgeView.Size = .medium) {
```

Comment:
can we add convinience method so that existing objc clients can stay the way it is, please?